index.bs

```diff
@@ -587,6 +587,44 @@ See also:
 * [[#secure-context]]
 * [[#consent]]
 
+<h3 id=text-formats>Make textual formats for people</h3>
```

Comment:
I think "make" reads a bit weird in this context. Maybe something like
```suggestion
<h3 id=text-formats>Textual formats should be usable by people</h3>
```
?

Reply:
s/Make/Design

```suggestion
<h3 id=text-formats>Design textual formats for people</h3>
```

We're using action statements throughout, and this is the best I had.  Yours ends up being a bit softer and not as imperative.  Don't we want them all to be "Do this", "Do that"?